man/adoc/bpftrace.adoc

```diff
@@ -977,6 +977,131 @@ The following snippets create a map with key signature `(int64, string[16])` and
 @[(pid, comm)]++
 ----
 
+==== Hygienic macros
+
+Hygienic macros (usually just "macros") provide a way for you to structure your script. They can be
+useful when you want to factor out code into smaller, more understandable parts. Or if you want to
+share code between probes.
+
+At a high level, macros can be thought of as semantic aware text replacement. They accept (optional)
+variable and map arguments - the body of the macro may only access maps and variables passed in through
```

Comment:
You say `maps and variables` here, but actually have an example with an expression `add_two(1)`. I think it would be good to nail that up front, e.g. maps and expressions.